listenbrainz/spark/handlers.py

```diff
@@ -203,3 +203,38 @@ def handle_recommendations(data):
                                                                similar_artist_recording_mbids)
 
     current_app.logger.debug("recommendation for {} inserted".format(musicbrainz_id))
+
+
+def handle_mapping(data):
```

Comment:
this name does not reflect what the function does

Reply:
handle_mapping_imported?

Reply:
Your doc string clearly names the key function of this method. Why is that not part of the name?

Reply:
`notify_mapping_import` ? 

Otherwise you tell me one :p

Reply:
Yep, that works!